circle.yml

```diff
@@ -61,8 +61,8 @@ executors:
   cy-doc:
     docker:
       - image: cypress/browsers:node16.5.0-chrome94-ff93
-    # by default, we use "small" to save on CI costs. bump on a per-job basis if needed.
-    resource_class: small
+    # by default, we use "medium" to balance performance + CI costs. bump or reduce on a per-job basis if needed.
+    resource_class: medium
```

Comment:
This is also the CircleCI default, so these lines could be removed:

```suggestion
```

Reply:
i wanted the line / explanation in there for clarity